### DISCUSSION TITLE
can i submit a report for stuttering youtube livestreams?

### DISCUSSION THREAD

**Discussion by EvilSamurai984** [uBlockOrigin/uAssets#23179] [Q&A]
I am not getting ads, just very bad performance in firefox some ublock updates.  I don't want to waste the devs time with uneeded or bad reports so let me know if this info may be valid and I will post the trouble shooting info, dumps from stats for nerds windows in firefox and chrome, as well as my own observations of how the issues behave when manifesting.

 Some of the livestreams only stutter in firefox after a ublock update. One of them is stuttering right now even with ublock disabled, but not stuttering in chrome. Would you like a data dump as well as my personal behavioral observations or is this the wrong place. Again, a stream that was fine on firefox will sometimes start stuttering after a ublock update and following the 5 steps did not fix the current one (the 5 steps fixed ones in the past)

**Comment by ghost** [comment]
Record performance profile (maybe one with "Media" preset?) using https://profiler.firefox.com/ and report here https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Untriaged
